johannes_mueller/entry_sphinx/Readme.md

```diff
@@ -0,0 +1,307 @@
+# Automated package documentation with Sphinx
+
+## Introduction
+We've all been there. We have written this wonderful piece of software that will essentially solve science's most pressing problems, the only *minor* thing left to do is propper documentation. And this, although it's only the lat few meters until the finish lines, is where good projects often turn into bad projects.
+
+<img src="https://user-images.githubusercontent.com/38459088/143226848-c0765ba7-43bc-4b9d-99ce-8426f2177cfa.jpg" width="400" height="400">
```

Comment:
Loved Patrick's images 🤣 
Can we use them?

Reply:
Hm...I would assume that typical memes are under CC license as they are naturally shared criss-cross the internet. The template is, for instance, available at meme-generator (https://imgflip.com/meme/183221332/PAtrick-Smart-Dumb), which doesn't bother with licenses at all ^^"

Reply:
Ok great! :)